app/tests/test_reporter.py

```diff
@@ -0,0 +1,93 @@
+import os
+from unittest.mock import MagicMock
+from pytest_mock.plugin import MockerFixture
+
+import pytest
+from pyaml_env import parse_config
+from typing import Dict, List
+
+from app.reporter import update_cost_reporter, get_current_and_previous_months_dates
+
+
+@pytest.fixture
+def aws_static_costs() -> Dict[str, int]:
+    return {"current_month": 700, "previous_month": 800}
+
+
+@pytest.fixture
+def aws_accounts_file() -> str:
+    aws_accounts_filename = "app/tests/manifests/test_accounts.yaml"
+    os.environ["AWS_COST_REPORTER_CONFIG"] = aws_accounts_filename
+
+    return aws_accounts_filename
+
+
+@pytest.fixture
+def aws_account_names(aws_accounts_file: str) -> List[str]:
+    return [
```

Comment:
this can be `parse_config(path=aws_accounts_file)["accounts"]` right?

Reply:
it will return a dict, I can modify the fixture to give `aws_accounts_data`

Reply:
parse_config(path=aws_accounts_file)["accounts"].keys()

Reply:
done